react/components/SKUSelector/utils/proptypes.js

```diff
@@ -1,47 +1,72 @@
 import PropTypes from 'prop-types'
 
-const skuPropType = PropTypes.shape({
-  /** Name of the SKU Item */
-  name: PropTypes.string.isRequired,
-  /** Images of the SKU item */
-  images: PropTypes.arrayOf(PropTypes.shape({
-    /** URL of source Image */
-    imageUrl: PropTypes.string.isRequired,
-    /** Brief description of the image */
-    imageLabel: PropTypes.string,
-  })).isRequired,
-  /** SkuID */
-  itemId: PropTypes.string.isRequired,
-  /** List of products specifications names */
-  variations: PropTypes.arrayOf(PropTypes.string),
+const skuPropType = variations => {
```

Comment:
A function that returns a propType seems strange. Here you could have created one `skuPropType` that uses `variationsForContainer` and other that uses `variationsParsed`. It would probably avoid some confusions.